tests/manage/monitoring/test_workload_with_distruptions.py

```diff
@@ -172,3 +178,168 @@ def test_workload_with_checksum_verify(
     ocp_pod = ocp.OCP(kind="Pod", namespace=project.namespace)
     sha1sum_output = ocp_pod.exec_oc_cmd(f"logs {pod_name}", out_yaml_format=False)
     logger.info("sha1sum output: %s", sha1sum_output)
+
+
+def get_mon_pod_by_pvc_name(pvc_name: str):
+    """
+    Function to get monitor pod by pvc_name label
+
+    Args:
+        pvc_name (str): name of the pvc the monitor pod is related to
+    """
+    mon_pod_ocp = (
+        ocp.OCP(
+            kind=constants.POD,
+            namespace=config.ENV_DATA["cluster_namespace"],
+            selector=f"pvc_name={pvc_name}",
+        )
+        .get()
+        .get("items")[0]
+    )
+    return Pod(**mon_pod_ocp)
```

Comment:
can we move this function to pod.py helper file?

Since this is a cherry pick PR, we can move with a follow up PR and probably only in the master branch

Reply:
addressed in https://github.com/red-hat-storage/ocs-ci/pull/8345 and cherry-picked